video-presentation.py

```diff
@@ -246,13 +286,19 @@ def find_slides_transitions(self, connector, host, secret_key, resource, masks=N
             self.logger.debug("Using advanced algorithm for finding slides. settings: %s", settings)
             results = self.slide_find_advanced(resource['local_paths'][0], masks=masks, **settings)
 
+        # Create and upload the compressed previews
+        mp4_preview, webm_preview = self.create_video_previews(resource['local_paths'][0])
+        mp4_preview_id = pyclowder.files.upload_preview(connector, host, secret_key, resource['id'], mp4_preview, {})
+        webm_preview_id = pyclowder.files.upload_preview(connector, host, secret_key, resource['id'], webm_preview, {})
+
         self.results = []
 
         slidesmeta = {
             'nrslides': 0,
             'listslides': [],
             'algorithm': self.algorithmsettings.get('algorithm', 'advanced'),
             'settings': settings,
+            'previews': {'mp4': mp4_preview_id, 'webm': webm_preview_id},
```

Comment:
you will have to update `extracter_info.json` with this field.

Reply:
Done